TestApp/Login/Sources/LoginViews/LoginScreen.swift

```diff
@@ -0,0 +1,61 @@
+import SwiftUI
+import Combine
+import Navigation
+import Login
+import UserStorage
+
+enum Route {
+    case dummySheet
+}
+
+@MainActor public final class LoginScreenViewModel: ObservableObject {
+
+    @Published var route: Route?
```

Comment:
Probably should be `Destination`?

I would understand the "Route" naming to be used only for the whole "path" (from the root to the leaf)

Reply:
Updated to `presented: PresentedDestination?` along with the rest of the project.